source-kinesis/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Build Stage
+################################################################################
+FROM golang:1.17-buster as builder
+
+WORKDIR /builder
+
+# Compile-time dependencies
+# RUN apt-get update \
+#  && apt-get install -y \
+#       ca-certificates \
+#  && apt-get clean
+
+# Download & compile dependencies. Doing this separately allows caching this
+# layer when no dependencies are updated.
+COPY go.* /builder/
+RUN go mod download
+
+# Build the connector projects we depend on
+COPY filesource /builder/filesource
+COPY source-kinesis /builder/source-kinesis
+
+# Run the unit tests
+RUN go test -v /builder/source-kinesis/...
+
+# Build the connector
+RUN go build -o /builder/connector -v /builder/source-kinesis/...
+
+
+# Runtime Stage
+################################################################################
+FROM gcr.io/distroless/base-debian10:debug
```

Comment:
This is a neat trick I wanted to point out. You can attach the `:debug` tag to any `distroless` image and it'll give you a busybox shell that you can `docker exec` into. :tada:

Reply:
Is this something we should do all the time, or something that can be used in a local checkout when debugging a connector ?

Reply:
No, not in the final release images. I just wanted to point it out to people here since I found out about it. It seemed like a good opportunity to share knowledge after I pushed this with the `:debug` tag still in there from when I was poking around. :upside_down_face: